### PR TITLE
Sweep volume-loss-weight 2/3/4 (targeting vol_pressure 2.08× AB-UPT gap)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

`volume_pressure` is the single worst-performing metric on tay with a **2.08× gap vs AB-UPT** (12.656% vs 6.08%). The current SOTA loss treats surface and volume predictions with equal weight (`surface_loss_weight=1.0`, `volume_loss_weight=1.0`). By upweighting the volume loss, we can force the model to allocate more representational capacity toward learning volume pressure gradients — the hardest component.

The volume branch predicts a single channel (volume pressure) while the surface branch predicts 4 channels (cp + tau_x/y/z). Equal raw loss weights may therefore effectively under-weight the volume prediction signal per sample. Increasing `--volume-loss-weight` from 1.0 toward 2.0–4.0 is a motivated, **zero-code-change** lever that has never been tested on tay.

**Experiment design:** 3-arm DDP8 sequential sweep over volume loss weights:
- **Arm A:** `--volume-loss-weight 2.0` (moderate upweight)
- **Arm B:** `--volume-loss-weight 3.0` (strong upweight)
- **Arm C:** `--volume-loss-weight 4.0` (aggressive upweight)

All other hyperparameters locked to SOTA stack from PR #232.

Note: there is a precedent on the tay branch — the very first baseline (PR #30) used `--volume-loss-weight 2.0` in the initial calibration config. That was on yi, not tay, and predates the current SOTA stack. We're now testing this lever with the fully-optimized Lion+EMA+warmup+heads=4 stack to see if the gap can be closed directly.

## Instructions

Run 3 arms **sequentially** on your 8 GPUs (DDP8 per arm). Use `--wandb_group tay-round13-vol-loss-weight` for all arms so runs are grouped in W&B.

### Arm A — volume-loss-weight=2.0
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent haku --wandb_group tay-round13-vol-loss-weight \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --lr-warmup-epochs 1 \
  --volume-loss-weight 2.0
```

### Arm B — volume-loss-weight=3.0
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent haku --wandb_group tay-round13-vol-loss-weight \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --lr-warmup-epochs 1 \
  --volume-loss-weight 3.0
```

### Arm C — volume-loss-weight=4.0
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent haku --wandb_group tay-round13-vol-loss-weight \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --lr-warmup-epochs 1 \
  --volume-loss-weight 4.0
```

**After all 3 arms complete**, post a comment with a results table showing per-arm best-epoch val_abupt, plus the full per-axis breakdown (surface_pressure, volume_pressure, wall_shear, tau_x, tau_y, tau_z), and W&B run IDs. Report test metrics from the best-val checkpoint of the best-performing arm.

Watch specifically whether surface metrics degrade as volume weight increases — this will tell us how much slack exists before hurting surface prediction.

## Baseline

**Current SOTA:** PR #232 (askeladd), W&B run `r8s2dtnq`, group `tay-round12-heads-4`

| Metric | Val (ep11 best) | Test | AB-UPT Target | Gap (×) |
|--------|----------------|------|---------------|---------|
| `abupt_axis_mean_rel_l2_pct` | **9.0650%** | 10.190% | — | — |
| `surface_pressure` | 5.703% | 5.461% | 3.82% | 1.43× |
| `volume_pressure` | 5.830% | 12.656% | 6.08% | **2.08× ← TARGET** |
| `wall_shear` | — | 9.910% | 7.29% | 1.36× |
| `tau_x` | — | 8.432% | 5.35% | 1.58× |
| `tau_y` | — | 11.952% | 3.65% | **3.27×** |
| `tau_z` | — | 12.447% | 3.63% | **3.43×** |

Beat target: `val_primary/abupt_axis_mean_rel_l2_pct < 9.0650%`

**Checkpoint selection metric:** `val_primary/abupt_axis_mean_rel_l2_pct` (lower is better)

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #319 because Closing before student starts — CURRENT_RESEARCH_STATE.md records 'Vol loss weight | CLOSED | 1.0 SOTA; 1.5 and 2.0 both negative'. Arm A (vol-loss-weight=2.0) is a known negative and the upweight direction was already falsified at two values. Reassigning haku to a fresh architectural axis (tau head reform / surface-tangent-frame projection). Sorry for the noise — cross-checked state file too late..
